letstest/README.md

```diff
@@ -19,35 +19,23 @@ This package is installed in the Certbot development environment that is
 created by following the instructions at
 https://certbot.eff.org/docs/contributing.html#running-a-local-copy-of-the-client.
 
-After activating that virtual environment, you can then configure AWS
-credentials and create a key by running:
-```
->aws configure --profile <profile name>
-[interactive: enter secrets for IAM role]
->aws ec2 create-key-pair --profile <profile name> --key-name <key name> --query 'KeyMaterial' --output text > whatever/path/you/want.pem
-```
-Note: whatever you pick for `<key name>` will be shown to other users with AWS access.
+These tests use the AWS SDK for Python (boto3) to manipulate EC2 instances.
+Before running the tests, you'll need to set up credentials by following the
+instructions at
+https://boto3.amazonaws.com/v1/documentation/api/latest/guide/quickstart.html#configuration.
+You will also want to create a `~/.aws/config` file setting the region for your
+profile to `us-east-1`.
 
-When prompted for a default region name, enter: `us-east-1`.
+Lastly, you will want to create a file on your system containing a trusted SSH key
+by following the instructions at
+https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/create-key-pairs.html.
 
 ## Usage
 To run tests, activate the virtual environment you created above and from this directory run:
 ```
 >letstest targets/targets.yaml /path/to/your/key.pem <profile name> scripts/<test to run>
 ```
 
-You can only run up to two tests at once. The following error is often indicative of there being too many AWS instances running on our account:
-```
-NameError: name 'instances' is not defined
```

Comment:
Did we fix this from popping up often? I don't remember. If not, we may still want a note about manually going into the console to shut down instances that didn't shut down on their own.

Reply:
I personally don't think this note is needed anymore largely because we only have one type of test farm test right now and we [only run that test on 5 machines](https://github.com/certbot/certbot/blob/21ef8e4332b41a7dd87ac83e548620cefe794d68/letstest/targets/targets.yaml). We used to have 4+ test scripts and run them on over 10 instances.

On top of that, [this text was written in 2017](https://github.com/certbot/certbot/pull/5091), we've done significant work on the test farm tests multiple times since then so it may not even be accurate anymore, https://github.com/certbot/certbot/pull/8536 made it so our instances self-destruct after an hour, and after poking around the EC2 console, I think our limit may be 200 instances (if not higher).

With all that in mind, what do you think? Do you think it's worth it for us to do anything other than just delete the text here?

Reply:
Ah perfect, it was probably #8536 that I was vaguely remembering. Agreed, no longer need this.